core/src/ast_builder/expr/function.rs

```diff
@@ -37,6 +37,8 @@ pub enum FunctionNode {
     Lcm(ExprNode, ExprNode),
     GenerateUuid,
     Repeat(ExprNode, ExprNode),
+    Lpad(ExprNode, ExprNode, Option<ExprNode>),
```

Comment:
How about going with the old structure? (in core/src/ast/funtion.rs)

```suggestion
    Lpad {
        expr: Expr,
        size: Expr,
        fill: Option<Expr>,
    }
```